ref/general/java-se.adoc

```diff
@@ -10,13 +10,13 @@
 :page-type: general
 = Java SE support
 
-Open Liberty requires a Java SE runtime in order to run. A Java SE runtime (also known as a Java Runtime Environment, or a Java Developer Kit), provides the tools to develop, run and monitor your Java application. Open Liberty can run on any Java SE 8 compatible runtime, or the Java SE 11 runtimes listed below. Due to behaviour and API changes between Java SE 8 and Java SE 11 it is possible that an application that runs on Open Liberty/Java SE 8 may not run on Open Liberty/Java SE 11, see the https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle Java SE 11 migration guide] for more details.
+Open Liberty requires a Java SE runtime in order to run. A Java SE runtime (also known as a Java Runtime Environment, or a Java Developer Kit), provides the tools to develop, run and monitor your Java application. Open Liberty can run on any Java SE 8, 11, or 12 compatible runtime listed below. Due to behaviour and API changes between Java SE 8 and Java SE 11 it is possible that an application that runs on Open Liberty/Java SE 8 may not run on Open Liberty/Java SE 11, see the https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle Java SE 11 migration guide] for more details.
 
 Open Liberty always recommends running the most recent release of Java available.
 
 == Java SE 8
 
-Any recent Java SE release from IBM, Oracle, AdoptOpenJDK will work. If using Oracle Java the minimum supported level is Java SE 8u25.
+Any recent Java SE release from IBM, Oracle, or AdoptOpenJDK will work. If using Oracle Java, the minimum supported level is Java SE 8u25.
```

Comment:
Change
"Any recent Java SE release from IBM, Oracle, or AdoptOpenJDK will work. If using Oracle Java, the minimum supported level is Java SE 8u25."
to
"Any recent Java SE release from IBM, Oracle, or AdoptOpenJDK works. With Oracle Java, the minimum supported level is Java SE 8u25."